src/components/confirmation-dialogs.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-import { Modal, Input, Checkbox } from 'antd';
+import { Modal, Input, Checkbox, Form } from 'antd';
 
 const confirm = Modal.confirm;
 const TextArea = Input.TextArea;
+const FormItem = Form.Item;
 
-export function confirmSectionDelete(t, section, onOk, onCancel = () => {}) {
+export function confirmSectionDelete(t, section, onOk, onCancel = () => { }) {
```

Comment:
I do not really care, I'm just wondering what made you add this space here (and in the other places ?

Reply:
The code editor does that automagically, I think there's something left in the settings, I'll try to find it.

Reply:
It's not a biggie, we just never have that space anywhere else.